src/test/java/core/basesyntax/service/filesoperation/FileWriteImplTest.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax.service.filesoperation;
+
+import static org.junit.Assert.assertTrue;
+
+import java.io.File;
+import org.junit.Before;
+import org.junit.Test;
+
+public class FileWriteImplTest {
+    private static final String FILE_PATH = "src/test/resources/report_file";
+    private static final String DATA = "fruit,quantity"
+                + System.lineSeparator() + "banana,152"
+                + System.lineSeparator() + "apple,90";
+    private FileWrite fileWrite;
+
+    @Before
+    public void setUp() {
+        fileWrite = new FileWriteImpl();
+    }
+
+    @Test
+    public void writeFilePath_Ok() {
+        fileWrite.write(FILE_PATH, DATA);
+        assertTrue(new File(FILE_PATH).exists());
+        assertTrue(new File(FILE_PATH).canRead());
```

Comment:
You're just testing that the file has been created and can be read, this doesn't guarantee that our method will work correctly. Create a separate read method that returns the read result and compare what it returns with what you wrote